Lib/platform.py

```diff
@@ -753,7 +753,7 @@ def _syscmd_uname(option, default=''):
         return default
     try:
         f = os.popen('uname %s 2> %s' % (option, DEV_NULL))
-    except (AttributeError, OSError):
+    except (AttributeError, OSError, ValueError):
```

Comment:
Looks suspicious.  Does os.popen() now return different exceptions?  I suspect you should fix os.popen() to trap ValueError and return OSError instead.  Otherwise everyone who uses os.popen() has to deal with the new exception case.